src/model.cpp

```diff
@@ -246,7 +245,7 @@ Real EmbedModel::train(shared_ptr<InternDataHandler> data,
     assert(end >= start);
     assert(end <= indices.end());
     bool amMaster = idx == 0;
-    int64_t elapsed;
+    //int64_t elapsed;
```

Comment:
why commenting this?

Reply:
If is unused, just delete it.

Reply:
yes, compiler said it was unused

Reply:
@jwijffels could you update the PR to just delete these lines?